modules/fundamental/src/sbom/endpoints/test.rs

```diff
@@ -14,6 +14,50 @@ use trustify_module_ingestor::{model::IngestResult, service::Format};
 use trustify_test_context::{TrustifyContext, call::CallService, document_bytes};
 use urlencoding::encode;
 
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn get_packages_sbom_by_query(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let app = caller(ctx).await?;
+    let id = ctx
+        .ingest_document("cyclonedx/openssl-3.0.7-18.el9_2.cdx_1.6.sbom.json")
+        .await?
+        .id
+        .to_string();
+
+    let result = query(&app, &id, "name~openssl-perl&Text~Apache").await;
+
+    assert!(
+        result
+            .items
+            .iter()
+            .any(|p| p.name == "openssl-perl"
+                && p.license_expression.as_deref() == Some("Apache-2.0"))
+    );
+    assert_eq!(result.total, 5);
+
+    let id = ctx
+        .ingest_document("spdx/OCP-TOOLS-4.11-RHEL-8.json")
+        .await?
+        .id
+        .to_string();
+
+    let result = query(&app, &id, "name=resolve&Text~MIT").await;
+    assert!(result.items.iter().any(|p| p.name == "resolve"
```

Comment:
I think we have some "contains subset" functions, which make it clearer what the tests are expecting. Unless they don't work, please use those instead.